protocol/messenger_handler.go

```diff
@@ -3015,9 +3015,11 @@ func (m *Messenger) isMessageAllowedFrom(publicKey string, chat *Chat) (bool, er
 	return contact.added(), nil
 }
 
-func (m *Messenger) updateUnviewedCounts(chat *Chat, mentionedOrReplied bool) {
-	chat.UnviewedMessagesCount++
-	if mentionedOrReplied {
+func (m *Messenger) updateUnviewedCounts(chat *Chat, message *common.Message) {
+	if message.Timestamp > chat.LastMessage.Timestamp+1000*60*2 {
```

Comment:
Maybe we should put that in a constant? It's possible we might reuse it and it would be clearer what it achieves.